### DISCUSSION TITLE
HIde Windows default flyouts when using ModernFlyouts

### DISCUSSION THREAD

**Discussion by hnavia** [ShankarBUS/ModernFlyouts#1151] [Q&A]
I'm using ModernFlyouts for a while but since last Windows Update 22H2, everytime I change the volume, Windows volume flyout appears as well.

![image](https://user-images.githubusercontent.com/69368602/208103596-8c0c6742-fe76-4dd0-99dd-c12de37395f1.png)

**Comment by headquarter8302** [comment]
This is a known tracked issue with a high priority, see #920
